setup.py

```diff
@@ -205,6 +205,7 @@ def get_git_sha() -> str:
         "vertica": ["sqlalchemy-vertica-python>=0.5.9, < 0.6"],
         "netezza": ["nzalchemy>=11.0.2"],
         "starrocks": ["starrocks>=1.0.0"],
+        "doris": ["pydoris>=1.0.0"],
```

Comment:
Could we include an upper bound of `2.0.0` to prevent potenitial breaking changes when the dependencies are bumped?

Reply:
OK I will modify it, thanks for your suggestion